src/backend/app/db/db_models.py

```diff
@@ -310,3 +310,19 @@ class DbUserProfile(Base):
     certificate_url = cast(str, Column(String, nullable=True))
     # drone registration certificate
     registration_certificate_url = cast(str, Column(String, nullable=True))
+
+
+class DbDroneAltitude(Base):
```

Comment:
It might be better to name the table like `drone_flight_height` or something like this.